app/controllers/users_controller.rb

```diff
@@ -1,4 +1,26 @@
 class UsersController < ApplicationController
   def show
   end
+
+  def new
+  end
+
+  def create
+    user = user_params
+    user[:name] = user[:name].downcase
+    new_user = User.create(user)
```

Comment:
Good catch on the passsssssssword problem we had here, thanks for updating this!